app/services/browser_support.rb

```diff
@@ -0,0 +1,58 @@
+class BrowserSupport
+  @cache = LruRedux::Cache.new(1_000)
+
+  BROWSERSLIST_TO_BROWSER_MAP = {
+    and_chr: ->(browser, version) { browser.android? && browser.chrome?(version) },
+    and_uc: ->(browser, version) { browser.android? && browser.uc_browser?(version) },
+    chrome: ->(browser, version) { browser.chrome?(version) },
+    edge: ->(browser, version) { browser.edge?(version) },
+    firefox: ->(browser, version) { browser.firefox?(version) },
+    ios_saf: ->(browser, version) { browser.ios? && browser.safari?(version) },
+    op_mini: ->(browser, version) { browser.opera_mini?(version) },
+    opera: ->(browser, version) { browser.opera?(version) },
+    safari: ->(browser, version) { browser.safari?(version) },
+    samsung: ->(browser, version) { browser.samsung_browser?(version) },
+  }.transform_values(&:to_proc).with_indifferent_access.freeze
```

Comment:
They're all procs now, so the transform is not adding anything
```suggestion
  }.with_indifferent_access.freeze
```

Reply:
> They're all procs now, so the transform is not adding anything

Oh, whoops, you're right 🤦

Reply:
> They're all procs now, so the transform is not adding anything

Removed in 38ee7a651.